src/test/java/org/jitsi/meet/test/base/Participant.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.base;
+
+import org.jitsi.meet.test.util.*;
+import org.openqa.selenium.*;
+
+import java.util.concurrent.*;
+
+/**
+ * The participant instance holding the {@link WebDriver}.
```

Comment:
It seems that many of the things here are web specific. Why isn't this class called WebParticipant ?

Reply:
Introduced WebParticipant.